data/Shashankzz0007.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Shashank Deshpande",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast! | DevOps | MERN Developer | Self-paced Full-Stack Developer",
+  "avatar": "https://github.com/Shashankzz0007.png",
+  "tags": ["Open Source", "javascript", "typescript", "Reactjs", "DevOps"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/shashankz007" },
+    { "platform": "github", "url": "https://github.com/Shashankzz0007" },
+    {
+      "platform": "linkedin",
+      "url": "https://www.linkedin.com/in/shashankd007"
+    },
+    {
+      "platform": "showcase",
+      "url": "https://www.showwcase.com/shashankzz0007"
+    }
+  ],
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Shashankzz0007",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/shashankz007",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/shashankd007",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on ShowCase",
+      "url": "https://www.showwcase.com/shashankzz0007",
+      "icon": "showcase"
+    },
+    {
+      "name": "Follow me on Discord",
+      "url": "https://discord.com/channels/@me",
```

Comment:
I don't think this will work. Also, Discord doesn't permit follows, I guess?

Reply:
Thanks for pointing, I'll change that.